crm_claim_rma/models/claim_line.py

```diff
@@ -0,0 +1,454 @@
+                   ('good', 'Product in good condition'),
+                   ('hidden', 'Product with hidden physical damage'),
+                   ],
+                  help="To describe the line product diagnosis")
+    claim_origin = fields.Selection(SUBJECT_LIST,
+                                    'Claim Subject',
+                                    required=True,
+                                    help="To describe the "
+                                    "line product problem")
+    product_id = fields.Many2one('product.product',
+                                 string='Product',
+                                 help="Returned product")
+    product_returned_quantity = \
+        fields.Float('Quantity', digits=(12, 2),
+                     help="Quantity of product returned")
+    unit_sale_price = fields.Float(digits=(12, 2),
+                                   help="Unit sale price of the product. "
+                                   "Auto filled if retrun done "
+                                   "by invoice selection. Be careful "
+                                   "and check the automatic "
+                                   "value as don't take into account "
+                                   "previous refunds, invoice "
+                                   "discount, can be for 0 if product "
+                                   "for free,...")
+    return_value = fields.Float(compute='_compute_line_total_amount',
+                                string='Total return',
+                                help="Quantity returned * Unit sold price",)
+    prodlot_id = fields.Many2one('stock.production.lot',
+                                 string='Serial/Lot number',
+                                 help="The serial/lot of "
+                                      "the returned product")
+    applicable_guarantee = fields.Selection([('us', 'Company'),
+                                             ('supplier', 'Supplier'),
+                                             ('brand', 'Brand manufacturer')],
+                                            'Warranty type')
+    guarantee_limit = fields.Date('Warranty limit',
+                                  readonly=True,
+                                  help="The warranty limit is "
+                                       "computed as: invoice date + warranty "
+                                       "defined on selected product.")
+    warning = fields.Selection(WARRANT_COMMENT,
+                               'Warranty',
+                               readonly=True,
+                               help="If warranty has expired")
+
+    @api.model
+    def get_warranty_return_partner(self):
+        return self.env['product.supplierinfo'].get_warranty_return_partner()
+
+    warranty_type = fields.Selection(get_warranty_return_partner,
+                                     readonly=True,
+                                     help="Who is in charge of the warranty "
```

Comment:
To use less lines of code, you can put first argument in the next line and then start with only one indentation more